app/models/manageiq/providers/inventory/persister.rb

```diff
@@ -105,6 +105,11 @@ def initialize_inventory_collections
     # can be implemented in a subclass
   end
 
+  def initialize_tag_mapper
+    @tag_mapper ||= ContainerLabelTagMapping.mapper
+    collections[:tags_to_resolve] = @tag_mapper.tags_to_resolve_collection
+  end
```

Comment:
```suggestion
  def tag_mapper
    @tag_mapper ||= ContainerLabelTagMapping.mapper.tap do |m|
      collections[:tags_to_resolve] = m.tags_to_resolve_collection
    end
  end
```

Since you are adding the accessor anyway, can we just make the tag_mapper method itself be a memoized methiod?

Reply:
Only issue I could see with that is then the inventory collection wouldn't be added until first reference to the`tag_mapper` where right now the providers initialize it with the rest of the inventory collections.  You could get someone trying to reference `persister.tags_to_resolve` but not call out to `persister.tag_mapper`